### PR TITLE
Level file mappings

### PR DESCRIPTION
**Pull request by YanisBft**
Small PR to map the new synchronous thing in 20w14a.

### CODE REVIEW

File: mappings/net/minecraft/world/storage/RegionFile.mapping [hunk 1]
```diff
@@ -11,9 +11,11 @@ CLASS net/minecraft/class_2861 net/minecraft/world/storage/RegionFile
 	METHOD <init> (Ljava/io/File;Ljava/io/File;Z)V
 		ARG 1 file
 		ARG 2 directory
+		ARG 3 synchronous
 	METHOD <init> (Ljava/nio/file/Path;Ljava/nio/file/Path;Lnet/minecraft/class_4486;Z)V
 		ARG 1 file
 		ARG 2 directory
+		ARG 4 synchronous
```
Comment (liach) [inline mappings/net/minecraft/world/storage/RegionFile.mapping]: This actually triggers [dsync](https://docs.oracle.com/javase/8/docs/api/java/nio/file/StandardOpenOption.html#DSYNC) standard open options flag than sync (only content is guaranteed, file metadata is not), dunno if this had better be `synchronous` or `dsync`

**Review by liach — APPROVED** [review]

File: build.gradle [hunk 2]
```diff
@@ -49,7 +49,7 @@ configurations {
 
 dependencies {
 	enigmaRuntime "net.fabricmc:stitch:0.4.3.71"
```
Comment (modmuss50) [inline build.gradle]: I would put this in a separate PR, as ideally other deps need updating as well for the asm 8 move

File: mappings/net/minecraft/world/storage/RegionFile.mapping (hunk 1 above)
Comment (YanisBft) [reply mappings/net/minecraft/world/storage/RegionFile.mapping]: updated

File: build.gradle (hunk 2 above)
Comment (YanisBft) [reply build.gradle]: reverted enigma update

**Review by modmuss50 — APPROVED** [review]
